Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css

```diff
@@ -0,0 +1,43 @@
+.actionButton {
+  box-sizing: border-box;
+  width: 6.7rem;
+  height: 6.8rem;
+  margin: 0;
+  padding: 1.2rem 0 0 0;
+  border: 1px solid gray;
+  background: firebrick;
+  color: white;
+  font-size: 2rem;
+  font-weight: bold;
+  text-align: center;
+}
+
```

Comment:
Also what I would do is have a general `btn` className. And then when you need to do something special to the button, either add an id to it or add a second className to it. 

i.e. `<button className="btn number">{props.number}</button>`

This allows for two classNames to be associated with the button. The btn will take care of the general overall style of all of the buttons. The second className will override anything you want to change from the original btn and then add stuff you want to add (i.e. change the color or the width).